index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
+      <span class="menu-button"></span>
+    </div>
+  </header>
+
+  <main>
+
+    <section class="wrapper hero">
+      <h1 class="hero-title">Hey There. I’m Alex.</h1>
```

Comment:
Great job on the title! The font styles are looking good. However, I noticed that the title is not matching the design requirement of being displayed in two lines. It's important to ensure that the layout and presentation of elements closely match the design specifications.

- [ ] Please make the necessary adjustments to the title layout to display it in two lines as per the design. This might involve modifying the CSS or adjusting the markup structure, depending on your implementation.

| Your  | Professional |
|---------------------|-----------------------|
| ![image](https://github.com/mahammad-mostafa/personal-portfolio-site/assets/55840999/5b5e70da-50bc-4a96-bd45-78a90d333161) | ![image](https://github.com/mahammad-mostafa/personal-portfolio-site/assets/55840999/20cdb346-c6b1-479d-bfa9-a0f400eb0d18) |